README.md

```diff
@@ -109,9 +109,9 @@ and execute the following four commands inside your docker container:
 
 ```bash
 ./manage.py add_oidc_client -n kerrokantasi-ui -t "id_token token" -u "http://localhost:8086/callback" "http://localhost:8086/silent-callback" -i https://api.hel.fi/auth/kerrokantasi-ui -m github -s dev
```

Comment:
For Kerrokantasi UI silent-callback would be `http://localhost:8086/silent-renew/`